projects/batfish-client/src/main/java/org/batfish/client/Command.java

```diff
@@ -173,7 +173,7 @@
                   "<analysis-name> <question-directory>",
                   "Initialize a new analysis for the container"));
       descs.put(INIT_CONTAINER, new Pair<>(
-            "[<container-name-prefix>]",
+            "<-setname><container-name> or [<container-name-prefix>]",
```

Comment:
this does not seem like the right syntax. 

`[-setname <container-name>] or [<container-name-prefix>]` seems closer. But probably it should be a multiline string with different options on different lines. Are there existing examples to mimic?